sql/moz-fx-data-shared-prod/monitor_backend/dataset_metadata.yaml

```diff
@@ -0,0 +1,12 @@
+friendly_name: monitor-backend
+description: |-
```

Comment:
A quirk of copying from generated metadata is you'll need to e.g. https://github.com/mozilla/bigquery-etl/compare/main...DSRE-1690-example#diff-adf3fe075eac21a06d3bd618d22bf4356b7e56d4011e0d7469599585d80332f3R2 to pass lint